lib/schedule/data.ex

```diff
@@ -384,18 +399,22 @@ defmodule Schedule.Data do
       blocks: blocks,
       calendar: gtfs_data.all_modes.calendar,
       runs: runs,
-      swings: swings
+      swings: swings,
+      version: gtfs_data.version
```

Comment:
**suggestion:** I know that right now the version is only being used for stops, and stop data comes entirely from GTFS, but I think it would be a good idea to make this a composite key of identifiers for both the GTFS as well as HASTUS data. The HASTUS data doesn't really have its own unique ID, but we could probably do something like just take the SHA-256 hash of the zip file.

Reply:
Good idea! We do have the hastus timestamp ~available (not currently a prop to `parse_files`, but could easily be passed). Do you have a preference between using the timestamp or a hash? I'm not sure if there is value to having `version` be human readable + directly reference visible data about the files.

Reply:
I don't think it's super useful to make it human-readable, just because it's easy to figure out from the `Schedule.Fetcher` logging when the application last pulled new schedule data.

Reply:
With that, updated in [7cd386f](https://github.com/mbta/skate/pull/2198/commits/7cd386f502d43031d2dcc63994605562fa20ed7b) to do a hash of all the files